src/main/java/Task.java

```diff
@@ -0,0 +1,70 @@
+import java.util.ArrayList;
+
+public class Task {
+    private String task;
+    private TaskStatus status;
```

Comment:
was wondering if using `isDone` or some other naming would make this clearer on whether the status of the task is done or not?